libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/route/RouteEngine.java

```diff
@@ -28,7 +30,7 @@ public RouteEngine(Callback engineCallback) {
     this.engineCallback = engineCallback;
   }
 
-  public void fetchRoute(Point origin, RouteProgress routeProgress) {
+  public void fetchRoute(Context context, Point origin, RouteProgress routeProgress) {
```

Comment:
Rather than passing the `Context` into `RouteEngine#fetchRoute`, let's add the `Locale` to the `RouteEngine` constructor or create an API like `RouteEngine#setLocale`